src/Microsoft.AspNetCore.Mvc.ViewFeatures/Controller.cs

```diff
@@ -102,7 +102,7 @@ public dynamic ViewBag
         [NonAction]
         public virtual ViewResult View()
         {
-            return View(viewName: null);
+            return View(viewName: ControllerContext.ActionDescriptor.ActionName);
```

Comment:
We need to try to avoid doing this here. The whole idea is that the controller is simply manufacturing the results; it's not adding much flavor to them. Then it's up to the result's execution logic to determine that no view name means to use the normalized action name.

This current code will likely break customer unit tests either due to a NullRef in CC.AD.AN, or due to it producing a non-null value.